view/frontend/web/css/magic.css

```diff
@@ -1,15 +1,23 @@
 #cart-occ-template {
```

Comment:
The way you introduce CSS changes is incorrect and does not allow proper customization.
Instead, your changes should be located in `web/css/_source/_magic.less` then included in the `_module.less`, compiled and minified on the go, instead of causing performance overhead of loading additional CSS files.